src/components/forms/transaction/SplitField.tsx

```diff
@@ -90,7 +90,7 @@ export default function SplitField({
         <span
           className="absolute px-1 py-2"
         >
-          {currencyToSymbol(account?.commodity?.mnemonic || '')}
+          {currencyToSymbol(account?.commodity?.mnemonic || '').slice(0, 3)}
```

Comment:
fixes layout for when IDs are too long. I.e.:

<img width="538" alt="Screenshot 2023-07-28 at 11 18 28 AM" src="https://github.com/maffin-io/maffin-app/assets/3578154/f6144baf-f5e5-4ac5-9239-123ad4aa7956">